docker/pebble/Dockerfile.windows

```diff
@@ -0,0 +1,18 @@
+FROM golang:1.11-nanoserver-sac2016 as builder
```

Comment:
`windows.Dockerfile` seems a better (for highlighting code at least) and more standard name than `Dockerfile.windows`.

Same thing for `Dockerfile.linux` -> `linux.Dockerfile`

Reply:
Indeed. Done.